lib/pengine/utils.c

```diff
@@ -2573,6 +2573,17 @@ pe__failed_probe_for_rsc(xmlNode *failed_ops, pe_resource_t *rsc, const char *no
 {
     const char *rsc_id = rsc->id;
 
+    if (rsc->variant == pe_clone) {
+#define VARIANT_CLONE 1
+#include "./variant.h"
+#undef VARIANT_CLONE
+
+        clone_variant_data_t *clone_data = NULL;
+        get_clone_variant_data(clone_data, rsc);
+
+        rsc_id = ID(clone_data->xml_obj_child);
+    }
```

Comment:
Rather than include the variant here, add a function for this to clone.c

Reply:
Fixed.